webapp/src/pages/Settings.tsx

```diff
@@ -77,6 +80,15 @@ type SubConfigKeys = keyof PickByValue<AzimuthConfig, object | null>;
 const CUSTOM_METRICS: string[] = ["Accuracy", "Precision", "Recall", "F1"];
 const ADDITIONAL_KWARGS_CUSTOM_METRICS = ["Precision", "Recall", "F1"];
 const SUPPORTED_LANGUAGES: SupportedLanguage[] = ["en", "fr"];
+const SUPPORTED_MODEL_CONTRACT: SupportedModelContract[] = [
```

Comment:
We should add an `S`:
```suggestion
const SUPPORTED_MODEL_CONTRACTS: SupportedModelContract[] = [
```